### DISCUSSION TITLE
cookie max age limited to 1 week?

### DISCUSSION THREAD

**Discussion by ac615223s5** [nuxt/nuxt#18904] [Questions] [answered]
When making a cookie with a long expiry date, it would get limited to one week. Does anyone know how to prevent this? I am using Firefox.
`
const x= useCookie("x", { maxAge: 1000000000});`
`
x.value="test";
`

![image](https://user-images.githubusercontent.com/33037196/217672963-ec617805-8aba-45af-a875-cd6857ea1403.png)

**Comment by AloisSeckar** [comment, marked as answer]
Max-value shouldn't be greater than **34560000**
https://httpwg.org/http-extensions/draft-ietf-httpbis-rfc6265bis.html#name-the-expires-attribute

Can you try with such value? Or little less? It might be Firefox is messing with too high value. Chrome looks like it can handle it:

![image](https://user-images.githubusercontent.com/7399922/217760363-590538af-adaf-44f1-8551-76f7825c71cd.png)

You may check what you'll get from this: https://stackblitz.com/edit/nuxt-starter-luzesx?file=app.vue

**Reply by ac615223s5** [reply]
Thanks, I lowered the value. It seems to work when first visiting the page but if the cookie is set after the page has loaded, it is limited to a week again

https://stackblitz.com/edit/nuxt-starter-8izvd6?file=app.vue

**Reply by AloisSeckar** [reply]
Only in your Firefox though, Chrome seems alright 
![image](https://user-images.githubusercontent.com/7399922/218021259-37b42b3c-37ca-4e1b-bdbd-643152a18ac0.png)

**Reply by AloisSeckar** [reply]
My fresh install of Firefox: 
![image](https://user-images.githubusercontent.com/7399922/218023013-3dc95252-1db4-408d-adae-10672c78c328.png)

**Reply by ac615223s5** [reply]
it seems to be a problem on my browser, thanks
